### ISSUE TITLE
[Short summary of the question]How to choose a model.

### ISSUE THREAD

**Issue by wangat** [cleanlab/cleanlab#685]
Thank you for your code and project. I used cleanlab to detect 4 classification task error labels (image was used for target detection, changed to classification task based on anchor). 8000 pictures checked manually were used, and the category was changed manually, resulting in 10% error. Then resnet50 was used to fine-tune 10-50 epochs under the pre-training weight (10 epochs were increased each time, a total of 5 experiments), and the cross verification was divided into 4 folds each time.

The final test results found that with the increase of epoch, the number of errors found increased, but only 85% of errors could be found at most, and the loss of the model was overfitted. It was also found that for the more difficult classification (two classes with small class spacing), only 60% of the assignment errors were found. In addition, it is found that some difficult samples are selected as wrong samples.

In another experiment, I used cleanlab to detect false labels for 4 classification tasks with data of over 200,000 (Sampling 10% data and manual inspection, the label accuracy reached 95%). Using resnet50, tweak 10epoch under the pre-training weight and finally find less than 1% error.

There is a lot of data to be processed, and it is generally used for target detection and segmentation tasks, and there is no specially trained classification model. How should I choose the corresponding model and training style?

Sorry for my poor English, thank you again.

**Comment by jwmueller** [comment]
The short answer is you should use the most accurate model you are able to produce without too much effort. The more accurate your model is the better find_label_issues will for you. Obviously making an accurate model is challenging and we cannot help you with that part via github comments nearly as well as other sources can help :) 

If you're still unsure what model to use for your application, you can also try out [Cleanlab Studio](https://cleanlab.ai/studio/?utm_source=github&utm_medium=docs&utm_campaign=clostostudio) which does all the modeling and detects the issues for you automatically, and provides a helpful interface to quickly review/fix them.

**Comment by wangat** [comment]
> The short answer is you should use the most accurate model you are able to produce without too much effort. The more accurate your model is the better find_label_issues will for you. Obviously making an accurate model is challenging and we cannot help you with that part via github comments nearly as well as other sources can help :)
> 
> If you're still unsure what model to use for your application, you can also try out [Cleanlab Studio](https://cleanlab.ai/studio/?utm_source=github&utm_medium=docs&utm_campaign=clostostudio) which does all the modeling and detects the issues for you automatically, and provides a helpful interface to quickly review/fix them.

Thank you for your reply. I understand how to handle this issue.
